### PR TITLE
Create sl.po

### PR DESCRIPTION
**Pull request by filmsi**
Created sl.po with Slovenian translation

### CODE REVIEW

**Comment by TingPing** [comment]
I don't understand why you did [the exact same thing](https://github.com/flatpak/xdg-desktop-portal-gtk/pull/487#issuecomment-2271668287). Please put both commits into one change.

**Comment by filmsi** [comment]
I will tell you why - because I am not a coder, just a translator.
You could enable translation via weblate or transifex and make it easier for localizers.

**Comment by filmsi** [comment]
Now it is done as you ordered.

File: po/sl.po [hunk 1]
```diff
@@ -0,0 +1,6238 @@
...
+#, fuzzy
+msgid "Arch:"
+msgstr "Arch A"
+
+#: app/flatpak-builtins-info.c:220 app/flatpak-builtins-info.c:262
+#: app/flatpak-builtins-remote-info.c:238
+#: app/flatpak-builtins-remote-info.c:273
+#, fuzzy
+msgid "Branch:"
+msgstr "Veja"
```
Comment (TingPing) [inline po/sl.po:1944]: Should these not keep the `:`, they are part of a formatted list:

```
Branch: ...
Arch: ...
```

File: po/sl.po [hunk 2]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-remote-info.c:292
+#, fuzzy
+msgid "Sdk:"
+msgstr ""
+"%PRODUCTNAME nudi vmesnik za programiranje aplikacij (API), ki omogoča "
+"nadziranje komponent $[officename] v raznih programskih jezikih ob uporabi "
+"razvojnega paketa (SDK) za $[officename]. Za več informacij o API "
+"$[officename] in o razvojnem paketu obiščite <link href=\"https://api."
+"libreoffice.org/\" name=\"api.libreoffice.org\">https://api.libreoffice.org</"
+"link>"
```
Comment (TingPing) [inline po/sl.po:1993]: This isn't right.

File: po/sl.po [hunk 3]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-info.c:267 app/flatpak-builtins-info.c:452
+#: app/flatpak-builtins-info.c:499
+msgid "Origin:"
+msgstr "Izvor:"
+
+#: app/flatpak-builtins-info.c:455 app/flatpak-builtins-info.c:506
+#, fuzzy
+msgid "Subpaths:"
+msgstr "Spoji podpoti"
```
Comment (TingPing) [inline po/sl.po:2067]: Same `:` comment.

File: po/sl.po [hunk 4]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-install.c:207
+msgid "Filename or uri must be specified"
+msgstr "Navesti je treba ime datoteke ali uri"
+
+#: app/flatpak-builtins-install.c:287
+#, fuzzy
+msgid "[LOCATION/REMOTE] [REF…] - Install applications or runtimes"
+msgstr ""
+"[LOKACIJA/DALJINEC] [REF...] - Namestite aplikacije ali izvajalne datoteke"
```
Comment (TingPing) [inline po/sl.po:2169]:
```suggestion
"[LOKACIJA/DALJINEC] [REF…] - Namestite aplikacije ali izvajalne datoteke"
```

File: po/sl.po [hunk 5]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-list.c:50 app/flatpak-builtins-remote-ls.c:55
+#, fuzzy
+msgid "List all refs (including locale/debug)"
+msgstr "Naštejte vse refs (vključno z lokalnimi ali debug)"
+
+#: app/flatpak-builtins-list.c:51 app/flatpak-builtins-remote-ls.c:56
+#, fuzzy
+msgid "List all applications using RUNTIME"
+msgstr "Na seznam vseh aplikacij z uporabo PROGRAMA RUNTIME"
```
Comment (TingPing) [inline po/sl.po:2254]: `PROGRAMA` shouldn't be capitalized, this refers to an argument named `RUNTIME`.

File: po/sl.po [hunk 6]
```diff
@@ -0,0 +1,6238 @@
...
+msgid "Default branch to use for this repository"
+msgstr "Privzeta veja, ki jo želite uporabiti za to skladišče"
+
+#: app/flatpak-builtins-build-update-repo.c:73
+#: app/flatpak-builtins-remote-add.c:79 app/flatpak-builtins-remote-modify.c:86
+#: app/flatpak-builtins-repo.c:710 app/flatpak-builtins-repo.c:711
+#: app/flatpak-builtins-run.c:69
+#, fuzzy
+msgid "BRANCH"
+msgstr "Veja"
```
Comment (TingPing) [inline po/sl.po:1055]: This is capitalized as it refers to an argument

File: po/sl.po [hunk 7]
```diff
@@ -0,0 +1,6238 @@
...
+#, fuzzy
+msgid "Extra data sizes of zero not supported"
+msgstr "Dodatne velikosti podatkov ničle niso podprte"
+
+#: app/flatpak-builtins-build-export.c:819
+#, fuzzy
+msgid ""
+"LOCATION DIRECTORY [BRANCH] - Create a repository from a build directory"
+msgstr ""
+"LOKACIJSKI IMENIK [PODRUŽNICA] - Ustvarite skladišče iz gradilnega imenika"
```
Comment (TingPing) [inline po/sl.po:562]: You use `PODRUŽNICA` here and `VEJA` elsewhere, they should match as they are the same argument.

File: po/sl.po [hunk 8]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-build-finish.c:51
+#, fuzzy
+msgid "Command to set"
+msgstr "Ukaz za nastavitev"
+
+#: app/flatpak-builtins-build-finish.c:51 app/flatpak-builtins-run.c:67
+#: app/flatpak-main.c:208
+#, fuzzy
+msgid "COMMAND"
+msgstr "Ukaz"
```
Comment (TingPing) [inline po/sl.po:620]: Same comment that this is capitalized as its an argument.

File: po/sl.po [hunk 9]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-build-finish.c:52
+#, fuzzy
+msgid "Flatpak version to require"
+msgstr "Flatpak različica zahtevati"
+
+#: app/flatpak-builtins-build-finish.c:52
+#, fuzzy
+msgid "MAJOR.MINOR.MICRO"
+msgstr "MAJOR. MLADOLETNIK. MIKRO"
```
Comment (TingPing) [inline po/sl.po:630]:
```suggestion
msgstr "MAJOR.MLADOLETNIK.MIKRO"
```

?

File: po/sl.po [hunk 10]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-build-finish.c:58
+#, fuzzy
+msgid "SDK"
+msgstr ""
+"%PRODUCTNAME nudi vmesnik za programiranje aplikacij (API), ki omogoča "
+"nadziranje komponent $[officename] v raznih programskih jezikih ob uporabi "
+"razvojnega paketa (SDK) za $[officename]. Za več informacij o API "
+"$[officename] in o razvojnem paketu obiščite <link href=\"https://api."
+"libreoffice.org/\" name=\"api.libreoffice.org\">https://api.libreoffice.org</"
+"link>"
```
Comment (TingPing) [inline po/sl.po:687]: Same.

File: po/sl.po [hunk 11]
```diff
@@ -0,0 +1,6238 @@
...
+#, fuzzy
+msgid "Change the runtime used for the app"
+msgstr "Spreminjanje časa zagona, uporabljenega za aplikacijo"
+
+#: app/flatpak-builtins-build-finish.c:59 app/flatpak-builtins-build-init.c:54
+#: app/flatpak-builtins-list.c:51 app/flatpak-builtins-remote-ls.c:56
+#: app/flatpak-builtins-run.c:71
+#, fuzzy
+msgid "RUNTIME"
+msgstr "Čas izvajanja:"
```
Comment (TingPing) [inline po/sl.po:699]: Seems wrong.

File: po/sl.po [hunk 12]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-build-import-bundle.c:45
+#, fuzzy
+msgid "Override the ref used for the imported bundle"
+msgstr "Preglasi sodnika, ki se uporablja za uvoženi snop"
+
+#: app/flatpak-builtins-build-import-bundle.c:45
+#, fuzzy
+msgid "REF"
+msgstr "Ref"
```
Comment (TingPing) [inline po/sl.po:809]:
```suggestion
msgstr "REF"
```

File: po/sl.po [hunk 13]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-build-init.c:55
+#, fuzzy
+msgid "APP"
+msgstr "Program"
+
+#: app/flatpak-builtins-build-init.c:56
+#, fuzzy
+msgid "Specify version for --base"
+msgstr "Navedite različico za --osnovno"
```
Comment (TingPing) [inline po/sl.po:865]:
```suggestion
msgstr "Navedite različico za --base"
```

File: po/sl.po [hunk 14]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-build-init.c:57
+#, fuzzy
+msgid "Include this base extension"
+msgstr "Vključi to osnovno razširitev"
+
+#: app/flatpak-builtins-build-init.c:57 app/flatpak-builtins-build-init.c:62
+#, fuzzy
+msgid "EXTENSION"
+msgstr "Pripona"
```
Comment (TingPing) [inline po/sl.po:879]: Same comments.

File: po/sl.po [hunk 15]
```diff
@@ -0,0 +1,6238 @@
...
+msgstr "VRSTA"
+
+#: app/flatpak-builtins-build-init.c:61
+msgid "Add a tag"
+msgstr "Dodaj značko"
+
+#: app/flatpak-builtins-build-init.c:61
+#, fuzzy
+msgid "TAG"
+msgstr "Značka"
```
Comment (TingPing) [inline po/sl.po:912]: Same.

File: po/sl.po [hunk 16]
```diff
@@ -0,0 +1,6238 @@
...
+#, fuzzy, c-format
+msgid "Requested extension %s not installed"
+msgstr "Zahtevana razširitev %s ni nameščena"
+
+#: app/flatpak-builtins-build-init.c:207
+#, fuzzy
+msgid ""
+"DIRECTORY APPNAME SDK RUNTIME [BRANCH] - Initialize a directory for building"
+msgstr ""
+"IMENIK APPNAME SDK RUNTIME [BRANCH] - Inicializacija imenika za gradnjo"
```
Comment (TingPing) [inline po/sl.po:944]: Inconsistent translation of `BRANCH`.

File: po/sl.po [hunk 17]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-install.c:70 app/flatpak-builtins-remote-info.c:55
+#: app/flatpak-builtins-uninstall.c:58 app/flatpak-builtins-update.c:64
+#, fuzzy
+msgid "Look for runtime with the specified name"
+msgstr "Poiščite čas zagona z navedenim imenom"
+
+#: app/flatpak-builtins-build-sign.c:65
+#, fuzzy
+msgid "LOCATION [ID [BRANCH]] - Sign an application or runtime"
+msgstr "LOKACIJA [ID [BRANCH]] - Podpišite aplikacijo ali čas zagona"
```
Comment (TingPing) [inline po/sl.po:984]: Same for `BRANCH`.

File: po/sl.po [hunk 18]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: app/flatpak-builtins-build-sign.c:72
+#: app/flatpak-builtins-build-update-repo.c:498
+#: app/flatpak-builtins-remote-add.c:313 app/flatpak-builtins-repo.c:735
+#, fuzzy
+msgid "LOCATION must be specified"
+msgstr "Določiti je treba lokacijo"
```
Comment (TingPing) [inline po/sl.po:991]: `LOCATION` is an argument.

File: po/sl.po [hunk 19]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-build-update-repo.c:69
+#, fuzzy
+msgid "A one-line comment for this repository"
+msgstr "Pripomba v eni vrstici za to skladišče"
+
+#: app/flatpak-builtins-build-update-repo.c:69
+#: app/flatpak-builtins-remote-add.c:75 app/flatpak-builtins-remote-modify.c:82
+#, fuzzy
+msgid "COMMENT"
+msgstr "Opomba"
```
Comment (TingPing) [inline po/sl.po:1022]: `COMMENT` is an argument.

File: po/sl.po [hunk 20]
```diff
@@ -0,0 +1,6238 @@
...
+#: app/flatpak-builtins-run.c:69
+#, fuzzy
+msgid "BRANCH"
+msgstr "Veja"
+
+#: app/flatpak-builtins-build-update-repo.c:74
+#: app/flatpak-builtins-remote-add.c:80 app/flatpak-builtins-remote-modify.c:87
+#, fuzzy
+msgid "COLLECTION-ID"
+msgstr "ID ZBIRKE"
```
Comment (TingPing) [inline po/sl.po:1061]: Is it possible to make this a single word, its a single argument.

File: po/sl.po [hunk 21]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1539
+#, fuzzy
+msgid "Add generic policy option"
+msgstr "Dodajanje možnosti splošnega pravilnika"
+
+#: common/flatpak-context.c:1539 common/flatpak-context.c:1540
+#, fuzzy
+msgid "SUBSYSTEM.KEY=VALUE"
+msgstr "PODSISTEMA. KEY=VREDNOST"
```
Comment (TingPing) [inline po/sl.po:4734]:
```suggestion
msgstr "PODSISTEMA.KEY=VREDNOST"
```

File: po/sl.po [hunk 22]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1326
+#, fuzzy, c-format
+msgid "Environment variable name must not contain '=': %s"
+msgstr "Ime spremenljivke okolja ne sme vsebovati '=': %s"
+
+#: common/flatpak-context.c:1439 common/flatpak-context.c:1447
+#, fuzzy
+msgid "--add-policy arguments must be in the form SUBSYSTEM.KEY=VALUE"
+msgstr "--argumenti dodatkov morajo biti v obrazcu SUBSYSTEM. KEY=VREDNOST"
```
Comment (TingPing) [inline po/sl.po:4562]:
```suggestion
msgstr "--argumenti dodatkov morajo biti v obrazcu SUBSYSTEM.KEY=VREDNOST"
```

File: po/sl.po [hunk 23]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1522
+#, fuzzy
+msgid "Expose socket to app"
+msgstr "Izpostavljanje vtičnice aplikaciji"
+
+#: common/flatpak-context.c:1522 common/flatpak-context.c:1523
+#, fuzzy
+msgid "SOCKET"
+msgstr "Vtičnica"
```
Comment (TingPing) [inline po/sl.po:4603]: Capitalized.

File: po/sl.po [hunk 24]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1524
+#, fuzzy
+msgid "Expose device to app"
+msgstr "Izpostavljanje naprave aplikaciji"
+
+#: common/flatpak-context.c:1524 common/flatpak-context.c:1525
+#, fuzzy
+msgid "DEVICE"
+msgstr "Naprava"
```
Comment (TingPing) [inline po/sl.po:4618]: Capitalized.

File: po/sl.po [hunk 25]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1526
+#, fuzzy
+msgid "Allow feature"
+msgstr "Dovoli funkcijo"
+
+#: common/flatpak-context.c:1526 common/flatpak-context.c:1527
+#, fuzzy
+msgid "FEATURE"
+msgstr "Nastavitve funkcije"
```
Comment (TingPing) [inline po/sl.po:4633]: Single word argument.

File: po/sl.po [hunk 26]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-context.c:1529
+#, fuzzy
+msgid "Don't expose filesystem to app"
+msgstr "Ne izpostavljajte datotečenega sistema aplikaciji"
+
+#: common/flatpak-context.c:1529
+#, fuzzy
+msgid "FILESYSTEM"
+msgstr "Seznam metod storitve FileSystem"
```
Comment (TingPing) [inline po/sl.po:4658]: Single word argument.

File: po/sl.po [hunk 27]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-dir.c:2687
+#, c-format
+msgid "System (%s) installation"
+msgstr "Namestitev sistema (%s)"
+
+#: common/flatpak-dir.c:2732
+#, fuzzy, c-format
+msgid "No overrides found for %s"
+msgstr "Za %s"
```
Comment (TingPing) [inline po/sl.po:4854]: I feel like some context is lost here.

File: po/sl.po [hunk 28]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-dir.c:10875
+#, fuzzy, c-format
+msgid "Pruning repo failed: %s"
+msgstr "Obrezovanje repo ni uspelo: %s"
+
+#: common/flatpak-dir.c:11043 common/flatpak-dir.c:11049
+#, fuzzy, c-format
+msgid "Failed to load filter '%s'"
+msgstr "Nalaganje filtra »%s« ni uspelo"
```
Comment (TingPing) [inline po/sl.po:5150]: `»«` is used here and `""` everywhere else?

File: po/sl.po [hunk 29]
```diff
@@ -0,0 +1,6238 @@
...
+msgstr "Neveljavna kontrolna vsota za indeksirane povzetke %s branje iz %s"
+
+#: common/flatpak-dir.c:11678
+#, fuzzy, c-format
+msgid ""
+"Remote listing for %s not available; server has no summary file. Check the "
+"URL passed to remote-add was valid."
+msgstr ""
+"Oddaljeni seznam za %s ni na voljo; strežnik nima datoteke s povzetjem. "
+"Preverite, ali je URL, ki je bil prenesti na oddaljeno dodajanje, veljaven."
```
Comment (TingPing) [inline po/sl.po:5184]: `remote-add` is a command and was lost in translation.

File: po/sl.po [hunk 30]
```diff
@@ -0,0 +1,6238 @@
...
+
+#: common/flatpak-run.c:1342
+#, fuzzy
+msgid "Failed to sync with dbus proxy"
+msgstr "Sinhronizacija z DBUS proxyjem ni uspela"
+
+#: common/flatpak-run.c:2144
+#, fuzzy
+msgid "No systemd user session available, cgroups not available"
+msgstr "Sistemska uporabniška seja ni na voljo, skupine niso na voljo"
```
Comment (TingPing) [inline po/sl.po:5595]: `systemd` is a projects name.

**Comment by filmsi** [comment]
Well, @TingPing, you are commenting on fuzzy translations - those are not finished and won't show in the build.
Please make comments about the finished translations, otherwise please check in this file.

Thank you.

**Comment by filmsi** [comment]
I am attaching an updated version of sl.po, now 33 % translated. Again, ignore the fuzzy strings, they are work in progress and do not build (=show in the builds).
[sl.po.zip](https://github.com/user-attachments/files/16664472/sl.po.zip)

**Comment by TingPing** [comment]
> those are not finished and won't show in the build.

Ok that is good to know.

Could you at least remove the clearly accidental copy pastes:

```
"%PRODUCTNAME nudi vmesnik za programiranje aplikacij (API), ki omogoča "
"nadziranje komponent $[officename] v raznih programskih jezikih ob uporabi "
"razvojnega paketa (SDK) za $[officename]. Za več informacij o API "
"$[officename] in o razvojnem paketu obiščite <link href=\"https://api."
"libreoffice.org/\" name=\"api.libreoffice.org\">https://api.libreoffice.org</"
"link>"
```

**Comment by filmsi** [comment]
The string you quote is not in the last file I sent.
Fuzzy strings come from translation memory of other projects I translate, like LibreOffice, Inkscape, Kdenilve, GIMP, Krita etc., so I really will not clean the fuzzy strings that do not make it into the build.

**Comment by filmsi** [comment]
Here you go, all fuzzy strings removed.
[sl.po.zip](https://github.com/user-attachments/files/16665509/sl.po.zip)

**Comment by filmsi** [comment]
What is going on here? Why are you blocking this translation?

**Comment by filmsi** [comment]
The po file seems to be checked in, but LINGUAS was not updated to include "sl".

**Comment by filmsi** [comment]
Not really. sl.po is not in main. Neither LINGUAS is updated.

**Pull request closed without merging**